packages/core/database/src/metadata/index.ts

```diff
@@ -40,12 +41,18 @@ export const createMetadata = (models: Model[] = []): Metadata => {
       throw new Error('The attribute "id" is reserved and cannot be used in a model');
     }
 
+    const documentIdAttribute: Meta['attributes'] =
+      model.modelType === 'contentType'
+        ? { documentID: { type: 'string', default: createId } }
```

Comment:
I'm wondering if we should introduce a new type for this attribute, to be able to differentiate it in the frontend (admin API)?

Reply:
Absolutely, I am having a meeting today with JS to locate where we could place the EntryId and DocId type in strapi types, will keep you updated!